gtfs/gtfs_utils/gtfs_utils/configuration.py

```diff
@@ -69,6 +69,7 @@ class Configuration:
     use_data_from_today: bool = True
     date_range: List[str] = None
     future_days_count: int = 0
+    max_gtfs_size_in_bytes: numbers.Number = float('inf')
```

Comment:
Did you make sure that it works even without adding this field in `config.json`?

Reply:
Yep. works fine :)